<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>647</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>140</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>801</width>
     <height>571</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">/**正常情况下样式**/
QPushButton{
    font: 10pt &quot;微软雅黑&quot;;
    color: white;
    background-color: transparent; /* 设置为透明 */
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton:hover{   
    color: #FFFFFF;
    background-color: #a2375a;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton:pressed, QPushButton:checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0  #273c75, stop:1 #202f55);
}

/**按钮失能情况下样式**/
QPushButton:disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
        </property>
        <property name="text">
         <string>生成随机素数</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">/**正常情况下样式**/
QPushButton{
    font: 10pt &quot;微软雅黑&quot;;
    color: white;
    background-color: transparent; /* 设置为透明 */
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton:hover{   
    color: #FFFFFF;
    background-color: #a2375a;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton:pressed, QPushButton:checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0  #273c75, stop:1 #202f55);
}

/**按钮失能情况下样式**/
QPushButton:disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
        </property>
        <property name="text">
         <string>获取私密密钥</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">/**正常情况下样式**/
QPushButton{
    font: 10pt &quot;微软雅黑&quot;;
    color: white;
    background-color: transparent; /* 设置为透明 */
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton:hover{   
    color: #FFFFFF;
    background-color: #a2375a;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton:pressed, QPushButton:checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0  #273c75, stop:1 #202f55);
}

/**按钮失能情况下样式**/
QPushButton:disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
        </property>
        <property name="text">
         <string>获取公开密钥</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="pushButton_4">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">/**正常情况下样式**/
QPushButton{
    font: 10pt &quot;微软雅黑&quot;;
    color: white;
    background-color: transparent; /* 设置为透明 */
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton:hover{   
    color: #FFFFFF;
    background-color: #a2375a;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton:pressed, QPushButton:checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0  #273c75, stop:1 #202f55);
}

/**按钮失能情况下样式**/
QPushButton:disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
        </property>
        <property name="text">
         <string>获取共享密钥</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignLeft">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>600</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; /* 设置为透明 */
    font: 10pt &quot;微软雅黑&quot;;
    color: white;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
